ecs-agent/gmsacredclient/credentialsfetcherclient.go

```diff
@@ -37,13 +42,7 @@ func GetGrpcClientConnection() (*grpc.ClientConn, error) {
 
 // getSocketAddress() returns the credentials-fetcher socket dir
 func getSocketAddress() (string, error) {
```

Comment:
Do we still need the `error` in function signature?

Reply:
we need the method back again